apps/backend/src/routes/home/+page.svelte

```diff
@@ -0,0 +1,389 @@
+            <th class="text-center">
+              <button on:click={showDetails} class="btn btn-ghost btn-xs hover:text-info">
+                details
+              </button>
+            </th>
+          </tr>
+          <!-- row 2 -->
+          <tr class="hover:bg-zinc-900 duration-80 border-b text-left py-2">
+            <td class="py-3"> Dexter's School </td>
+            <td class="py-3"> Dezidawk </td>
+            <td class="py-3"> Aug 26, 2023 me </td>
+            <td class="py-3"> 111 KB </td>
+            <th class="text-center py-3">
+              <button on:click={showDetails} class="btn btn-ghost btn-xs hover:text-info">
+                details
+              </button>
+            </th>
+          </tr>
+          <!-- row 3 -->
+          <tr class="hover:bg-zinc-900 duration-80 border-b text-left py-2">
+            <td class="py-3"> Dr Receipts Pitch </td>
+            <td class="py-3"> Dezidawk </td>
+            <td class="py-3">Jan 20, 2022 me </td>
+            <td class="py-3"> 4 KB </td>
+            <th class="text-center py-3">
+              <button on:click={showDetails} class="btn btn-ghost btn-xs hover:text-info">
+                details
+              </button>
+            </th>
+          </tr>
+          <!-- row 4 -->
+          <tr class="hover:bg-zinc-900 duration-80 border-b text-left py-2">
+            <td class="py-3"> Infoseceurity Project </td>
+            <td class="py-3"> Dezidawk </td>
+            <td class="py-3"> April 5, 2023 me </td>
+            <td class="py-3"> 15 KB </td>
+            <th class="text-center py-3">
+              <button on:click={showDetails} class="btn btn-ghost btn-xs hover:text-info">
+                details
+              </button>
+            </th>
+          </tr>
+        </tbody>
+      </table>
+    </div>
+  </div>
+
+  <!-- Info Card -->
+  {#if showInfoCard}
+    <div
+      in:fade={{ duration: 300 }}
+      out:fade={{ duration: 300 }}
+      class="w-96 max-w-7xl h-screen p-5 rounded-md border border-white">
+      <div class="flex items-center pb-2">
+        <h1 class="flex-grow font-semibold">Example File Name.pdf</h1>
+        <button
+          on:click={hideDetails}
+          class="btn font-semibold normal-case rounded-full border-none bg-transparent">
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            class="h-6 w-6"
+            fill="none"
+            viewBox="0 0 24 24"
+            stroke="currentColor"
+            ><path
+              stroke-linecap="round"
+              stroke-linejoin="round"
+              stroke-width="2"
+              d="M6 18L18 6M6 6l12 12" /></svg>
+        </button>
+      </div>
+      <hr />
+      <div class="pt-5">
+        <h2 class="pb-2 font-semibold">File Details</h2>
+
+        <!-- File Access Section -->
+        <div class="py-2">
+          <button
+            class="
+            btn btn-outline btn-info rounded-full font-semibold normal-case px-6"
```

Comment:
If these 2 lines can be combined into one, why not make it one?

Reply:
Okay limpeh combine alr